static/app/views/settings/account/notifications/notificationSettingsByType.tsx

```diff
@@ -26,7 +26,7 @@ import {
   SupportedProviders,
 } from './constants';
 import {ACCOUNT_NOTIFICATION_FIELDS} from './fields';
-import {NOTIFICATION_SETTING_FIELDS_V2, QUOTA_FIELDS} from './fields2';
+import {NOTIFICATION_SETTING_FIELDS, QUOTA_FIELDS} from './fields2';
```

Comment:
whats going on with fields2 is that something we can cleanup

Reply:
@scttcper nah we can't unfortunately